app/javascript/app/reducers.js

```diff
@@ -28,6 +28,7 @@ import * as emissionsProvider from 'providers/emissions-provider';
 import * as dataExplorerProvider from 'providers/data-explorer-provider';
 import * as lawsAndPoliciesProvider from 'providers/laws-and-policies-provider';
 import * as agricultureCountriesContextsProvider from 'providers/agriculture-countries-context-provider';
+import * as agricultureEmissionsProvider from 'providers/agriculture-emissions-provider';
```

Comment:
note how it is different from the import in styles; in this case you are using it correctly - you're importing the implicitly exported vars from the agriculture provider.